core/src/main/java/jenkins/model/logging/LogHandler.java

```diff
@@ -0,0 +1,55 @@
+package jenkins.model.logging;
+
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.Beta;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import java.io.Serializable;
+
+/**
+ * Object which operates with {@link Loggable} items.
+ * @author Oleg Nenashev
+ * @since TODO
+ */
+@ExportedBean
+@Restricted(Beta.class)
+public abstract class LogHandler {
+
+    protected transient Loggable loggable;
+
+    public LogHandler(@Nonnull Loggable loggable) {
+        this.loggable = loggable;
+    }
+
+    @Exported
+    public String getId() {
```

Comment:
What is this used for? I would drop it. Stapler exporting already specifies `_class` where required.